features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/utils/ImageWireframeHelper.kt

```diff
@@ -82,18 +84,25 @@ interface ImageWireframeHelper {
         clipping: MobileSegment.WireframeClip? = null,
         shapeStyle: MobileSegment.ShapeStyle? = null,
         border: MobileSegment.ShapeBorder? = null,
-        prefix: String? = DRAWABLE_CHILD_NAME
+        prefix: String? = DRAWABLE_CHILD_NAME,
+        resourceIdCacheKey: String?
```

Comment:
are we not assigning here the default `null` value on purpose? other nullable args in this function have it.